cli/containerdutils/types.go

```diff
@@ -0,0 +1,144 @@
+package containerdutils
+
+import (
+	"context"
+
+	"github.com/containerd/containerd"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
+	ver "github.com/hashicorp/go-version"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+var (
+	// ContainerdSockPath is the default containerd socket
+	ContainerdSockPath = "/run/containerd/containerd.sock"
+
+	// DefaultDockerRootDir is the default daemon data dir
+	DefaultDockerRootDir = "/var/lib/docker"
+
+	// EngineNamespace is the default namespace where dockerd is run on containerd
+	EngineNamespace = "docker"
+
+	// EngineContainerName is the name given to the dockerd container
+	EngineContainerName = "dockerd"
+
+	// CommunityEngineImage is the repo name for the community engine
+	CommunityEngineImage = "engine-community"
+
+	// EnterpriseEngineImage is the repo name for the enterprise engine
+	EnterpriseEngineImage = "engine-enterprise"
+
+	// EngineSpec is the default container spec for the docker engine
+	EngineSpec = specs.Spec{
```

Comment:
Seems like this should be a function to get the default spec (since that's the only way it could be immutable)...
This will probably work, though.